Gulpfile.js

```diff
@@ -143,3 +144,48 @@ gulp.task('saucelabs', ['lint', 'build'], function (done) {
         .then(testsDone)
         .catch(testsDone);
 });
+
+gulp.task('cli', ['lint', 'build'], function (done) {
```

Comment:
https://devexpress.github.io/testcafe/documentation/recipes/running-tests-in-firefox-and-chrome-using-travis-ci.html